README.md

```diff
@@ -1,42 +1,42 @@
-# VML 
-[![Build Status](https://travis-ci.com/Crown421/VML.jl.svg?branch=master)](https://travis-ci.com/Crown421/VML.jl)
-[![Build status](https://ci.appveyor.com/api/projects/status/btdduqfsxux8fhsr?svg=true)](https://ci.appveyor.com/project/Crown421/vml-jl)
+# IntelVectorMath
```

Comment:
```suggestion
# IntelVectorMath.jl (formerly VML.jl)
```
I like the name change, but suggest making it very clear that this is the same package under a new name.

Would also suggest spelling this out in the first paragraph, perhaps like so:
```
This package provides bindings to the Intel MKL [Vector Mathematics Functions](https://software.intel.com/en-us/node/521751).
This is often substantially faster than broadcasting Julia's built-in functions, 
especially when applying a transcendental function over a large array. 
Until Julia 0.6 the package was registered as `VML.jl`.
```

Reply:
I have included your suggestions and a few more changes.

Reply:
Thanks!